pkg/sources/git/git.go

```diff
@@ -419,6 +419,8 @@ func executeClone(ctx context.Context, params cloneParams) (*git.Repository, err
 		"clone",
 		cloneURL.String(),
 		params.clonePath,
+		"-c",
+		"remote.origin.fetch=+refs/*:refs/remotes/origin/*",
```

Comment:
question: Do you foresee any situation where someone _wouldn't_ want this? If so, could we consider making it configurable?